src/cartogram_info/write_geojson.cpp

```diff
@@ -153,7 +153,7 @@ void CartogramInfo::json_to_geojson(
     // TODO: THE NEXT LINE CREATES A FIELD WITH VALUE null UNLESS THE
     //       INPUT GEOJSON CONTAINED A FIELD CALLED id. THIS MAY BE NEEDED BY
     //       cartogram_web, BUT, IN THE LONG RUN, WE WANT TO GET RID OF IT.
-    new_json["features"][i]["id"] = old_json["features"][index]["id"];
+    //new_json["features"][i]["id"] = old_json["features"][index]["id"];
```

Comment:
Line 156 along with the TODO comment can be removed. The cartogram id that is needed for the web is inside features->properties (automatically handled by lines 157-158).